### DISCUSSION TITLE
TeamTalk no recognize integrated camera - Lenovo P52S

### DISCUSSION THREAD

**Discussion by KP4AJ** [BearWare/TeamTalk5#1533] [General]
Hi:
Thanks for this wonderful software.  I am having a problem where TeamTalk is not able to initialize the Lenovo p52s integrated camera.   Camera works fine with other apps including Microsoft Teams.  I have verify Privacy Settings and all are clear so dont sure what else to try.  

The only software affected is TeamTalk.  I have tried uninstalling other video apps and even created a new profile in Windows.  Nothing working so far.  

I have and old Think Pad 430S and in that laptop the camera works with TeamTalk 5.11.  Noticed the drivers identifies as USB device while the P52S identifies as Integrated Camera, 

Thanks in Advance!

Edfel

**Comment by lbk2907** [comment]
I'm using Microsoft Surface Pro 7. TeamTalk also didn't recognise the
camera. I got the dialog that said "Fail to start video" something like
that. I forgot the exact dialog.

**Comment by KP4AJ** [comment]
Hi:
I tried this but no luck here.  https://www.auslogics.com/en/articles/fix-integrated-built-in-webcam-in-windows-10/

**Comment by KP4AJ** [comment]
Hi:
Found this:
https://github.com/BearWare/TeamTalk5/issues/584

and this other
https://github.com/BearWare/TeamTalk5/issues/780

But seems no solution so far.

**Comment by KP4AJ** [comment]
Hi All:
Solved.  I observed Lenovo 430 camera use YUY2 so  tried same with the Lenovo P52S and after pressing the presets was able to select different options.  Not sure If I did other change BUT I have video!!  

Problem new Laptop defaults to I420 not aware the were presets in addition to custom video format under YUY2. Was able to use several.   

Happy camper here!

**Comment by KP4AJ** [comment]
Hi:
Sorry for the mess in the photo.  I am a Ham Radio operator using TeamTalk within AREDN Mesh in Puerto Rico.  Experimental.  Running it Ubuntu 22.04.  

Keep the good job!! God bless.

Regards and 73'!

Edfel
KP4AJ

**Comment by lbk2907** [comment]
Thanks.
